Index: horizon-2012.2/run_tests.sh
===================================================================
--- horizon-2012.2.orig/run_tests.sh	2012-07-25 17:42:38.186508741 -0700
+++ horizon-2012.2/run_tests.sh	2012-07-25 17:42:42.614508546 -0700
@@ -265,22 +265,22 @@
 
   echo "Running Horizon application tests"
   export NOSE_XUNIT_FILE=horizon/nosetests.xml
-  ${command_wrapper} coverage erase
-  ${command_wrapper} coverage run -p $root/manage.py test horizon --settings=horizon.tests.testsettings $testargs
+  ${command_wrapper} python-coverage erase
+  ${command_wrapper} python-coverage run -p $root/manage.py test horizon --settings=horizon.tests.testsettings $testargs
   # get results of the Horizon tests
   HORIZON_RESULT=$?
 
   echo "Running openstack_dashboard tests"
   export NOSE_XUNIT_FILE=openstack_dashboard/nosetests.xml
-  ${command_wrapper} coverage run -p $root/manage.py test openstack_dashboard --settings=openstack_dashboard.test.settings $testargs
+  ${command_wrapper} python-coverage run -p $root/manage.py test openstack_dashboard --settings=openstack_dashboard.test.settings $testargs
   # get results of the openstack_dashboard tests
   DASHBOARD_RESULT=$?
 
   if [ $with_coverage -eq 1 ]; then
     echo "Generating coverage reports"
-    ${command_wrapper} coverage combine
-    ${command_wrapper} coverage xml -i --omit='/usr*,setup.py,*egg*,.venv/*'
-    ${command_wrapper} coverage html -i --omit='/usr*,setup.py,*egg*,.venv/*' -d reports
+    ${command_wrapper} python-coverage combine
+    ${command_wrapper} python-coverage xml -i --omit='/usr*,setup.py,*egg*,.venv/*'
+    ${command_wrapper} python-coverage html -i --omit='/usr*,setup.py,*egg*,.venv/*' -d reports
   fi
   # Remove the leftover coverage files from the -p flag earlier.
   rm -f .coverage.*
