diff -Naurp horizon-2012.2.orig/run_tests.sh horizon-2012.2/run_tests.sh
--- horizon-2012.2.orig/run_tests.sh	2012-05-03 19:47:00.000000000 -0400
+++ horizon-2012.2/run_tests.sh	2012-05-04 09:44:49.136820448 -0400
@@ -272,8 +272,8 @@ function run_tests {
   sanity_check
 
   echo "Running Horizon application tests"
-  ${command_wrapper} coverage erase
-  ${command_wrapper} coverage run -p $root/manage.py test horizon --settings=horizon.tests.testsettings $testargs
+  ${command_wrapper} python-coverage erase
+  ${command_wrapper} python-coverage run -p $root/manage.py test horizon --settings=horizon.tests.testsettings $testargs
   # get results of the Horizon tests
   HORIZON_RESULT=$?
 
@@ -281,15 +281,15 @@ function run_tests {
   if [ $selenium -eq 1 ]; then
     export WITH_SELENIUM=1
   fi
-  ${command_wrapper} coverage run -p $root/manage.py test openstack_dashboard --settings=openstack_dashboard.test.settings $testargs
+  ${command_wrapper} python-coverage run -p $root/manage.py test openstack_dashboard --settings=openstack_dashboard.test.settings $testargs
   # get results of the openstack_dashboard tests
   DASHBOARD_RESULT=$?
 
   if [ $with_coverage -eq 1 ]; then
     echo "Generating coverage reports"
-    ${command_wrapper} coverage combine
-    ${command_wrapper} coverage xml -i --omit='/usr*,setup.py,*egg*,.venv/*'
-    ${command_wrapper} coverage html -i --omit='/usr*,setup.py,*egg*,.venv/*' -d reports
+    ${command_wrapper} python-coverage combine
+    ${command_wrapper} python-coverage xml -i --omit='/usr*,setup.py,*egg*,.venv/*'
+    ${command_wrapper} python-coverage html -i --omit='/usr*,setup.py,*egg*,.venv/*' -d reports
   fi
   # Remove the leftover coverage files from the -p flag earlier.
   rm -f .coverage.*
