diff --git a/MANIFEST.in b/MANIFEST.in
index 4228b9f..4946257 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,6 +1,6 @@
-recursive-include doc *.py *.rst *.scss *.js *.html *.conf *.jpg *.gif *.png
-recursive-include horizon *.html *.scss *.js *.csv *.template *.tmpl *.mo *.po
-recursive-include openstack_dashboard *.html *.js *.scss *.mo *.po *.example *.eot *.svg *.ttf *.woff *.png *.ico *.wsgi *.gif *.csv *.template
+recursive-include doc *.py *.rst *.scss *.js *.html *.conf *.jpg *.gif *.png *.css
+recursive-include horizon *.html *.scss *.js *.csv *.template *.tmpl *.mo *.po *.css
+recursive-include openstack_dashboard *.html *.js *.scss *.mo *.po *.example *.eot *.svg *.ttf *.woff *.png *.ico *.wsgi *.gif *.csv *.template *.css
 recursive-include tools *.py *.sh
 
 include AUTHORS
