diff -Naurp horizon-2012.1.orig/MANIFEST.inf horizon-2012.1/MANIFEST.inf
--- horizon-2012.1.orig/MANIFEST.inf	1969-12-31 19:00:00.000000000 -0500
+++ horizon-2012.1/MANIFEST.inf	2012-03-02 12:01:13.142846863 -0500
@@ -0,0 +1 @@
+recursive-include horizon *.html *.js *.csv *.template *.mo
diff -Naurp horizon-2012.1.orig/setup.py horizon-2012.1/setup.py
--- horizon-2012.1.orig/setup.py	2012-03-01 05:01:06.000000000 -0500
+++ horizon-2012.1/setup.py	2012-03-02 12:01:51.870848117 -0500
@@ -36,6 +36,7 @@ setup(name="horizon",
       author='Devin Carlen',
       author_email='devin.carlen@gmail.com',
       packages=find_packages(),
+      include+package_data=True,
       package_data={'horizon': [s[len('horizon/'):] for s in
                                 findall('horizon/templates') \
                                 + findall('horizon/dashboards/nova/templates') \
