Index: horizon/openstack_dashboard/local/local_settings.py.example
===================================================================
--- horizon.orig/openstack_dashboard/local/local_settings.py.example	2012-06-20 15:13:24.780980297 -0700
+++ horizon/openstack_dashboard/local/local_settings.py.example	2012-06-20 15:13:51.544979315 -0700
@@ -2,7 +2,7 @@
 
 from django.utils.translation import ugettext_lazy as _
 
-DEBUG = True
+DEBUG = False
 TEMPLATE_DEBUG = DEBUG
 
 # Set SSL proxy settings:
