diff -Naurp horizon-2012.2.orig/openstack_dashboard/local/local_settings.py.example horizon-2012.2/openstack_dashboard/local/local_settings.py.example
--- horizon-2012.2.orig/openstack_dashboard/local/local_settings.py.example	2012-05-30 18:38:58.000000000 -0400
+++ horizon-2012.2/openstack_dashboard/local/local_settings.py.example	2012-05-31 14:14:37.080889312 -0400
@@ -2,7 +2,7 @@ import os
 
 from django.utils.translation import ugettext_lazy as _
 
-DEBUG = True
+DEBUG = False
 TEMPLATE_DEBUG = DEBUG
 PROD = False
 
