diff -Naurp horizon-2013.2.a96.gd17fd51.orig/openstack_dashboard/local/local_settings.py.example horizon-2013.2.a96.gd17fd51/openstack_dashboard/local/local_settings.py.example
--- horizon-2013.2.a96.gd17fd51.orig/openstack_dashboard/local/local_settings.py.example	2013-05-21 09:33:32.000000000 -0500
+++ horizon-2013.2.a96.gd17fd51/openstack_dashboard/local/local_settings.py.example	2013-05-21 11:34:34.426566745 -0500
@@ -4,7 +4,7 @@ from django.utils.translation import uge
 
 from openstack_dashboard import exceptions
 
-DEBUG = True
+DEBUG = False
 TEMPLATE_DEBUG = DEBUG
 
 # Required for Django 1.5.
@@ -90,19 +90,29 @@ SECRET_KEY = secret_key.generate_or_read
 # We recommend you use memcached for development; otherwise after every reload
 # of the django development server, you will have to login again. To use
 # memcached set CACHES to something like
-# CACHES = {
-#    'default': {
-#        'BACKEND' : 'django.core.cache.backends.memcached.MemcachedCache',
-#        'LOCATION' : '127.0.0.1:11211',
-#    }
-#}
-
 CACHES = {
-    'default': {
-        'BACKEND' : 'django.core.cache.backends.locmem.LocMemCache'
-    }
+   'default': {
+       'BACKEND' : 'django.core.cache.backends.memcached.MemcachedCache',
+       'LOCATION' : '127.0.0.1:11211',
+   }
 }
 
+# Enable the Ubuntu theme if it is present.
+try:
+    from ubuntu_theme import *
+except ImportError:
+    pass
+
+# Default Ubuntu apache configuration uses /horizon as the application root.
+# Configure auth redirects here accordingly.
+LOGIN_URL='/horizon/auth/login/'
+LOGIN_REDIRECT_URL='/horizon'
+
+# The Ubuntu package includes pre-compressed JS and compiled CSS to allow
+# offline compression by default.  To enable online compression, install
+# the node-less package and enable the following option.
+COMPRESS_OFFLINE = True
+
 # Send email to the console by default
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 # Or send them to /dev/null
