diff --git a/horizon/api/nova.py b/horizon/api/nova.py
index 3bf209f..980f378 100644
--- a/horizon/api/nova.py
+++ b/horizon/api/nova.py
@@ -513,7 +513,8 @@ def volume_list(request, search_opts=None):
     To see all volumes in the cloud as an admin you can pass in a special
     search option: {'all_tenants': 1}
     """
-    return cinderclient(request).volumes.list(search_opts=search_opts)
+    return []
+#    return cinderclient(request).volumes.list(search_opts=search_opts)
 
 
 def volume_get(request, volume_id):
@@ -565,7 +566,8 @@ def volume_snapshot_get(request, snapshot_id):
 
 
 def volume_snapshot_list(request):
-    return cinderclient(request).volume_snapshots.list()
+    return []
+#    return cinderclient(request).volume_snapshots.list()
 
 
 def volume_snapshot_create(request, volume_id, name, description):
