diff --git a/horizon/api/nova.py b/horizon/api/nova.py
index 3bf209f..d89ab76 100644
--- a/horizon/api/nova.py
+++ b/horizon/api/nova.py
@@ -249,12 +249,14 @@ def flavor_list(request):
 
 def tenant_floating_ip_list(request):
     """Fetches a list of all floating ips."""
-    return novaclient(request).floating_ips.list()
+    return []
+    #return novaclient(request).floating_ips.list()
 
 
 def floating_ip_pools_list(request):
     """Fetches a list of all floating ip pools."""
-    return novaclient(request).floating_ip_pools.list()
+    return []
+    #return novaclient(request).floating_ip_pools.list()
 
 
 def tenant_floating_ip_get(request, floating_ip_id):
